### PR TITLE
bpo-39719: remove softspace from tempfile.SpooledTemporaryFile

### PR DESCRIPTION
**Pull request by hauntsaninja**
https://bugs.python.org/issue39719

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2020-02-21-22-35-57.bpo-39719.2jLy1C.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove :meth:`tempfile.SpooledTemporaryFile.softspace` as files no longer have the ``softspace`` attribute Python 3.
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Library/2020-02-21-22-35-57.bpo-39719.2jLy1C.rst]:
```suggestion
Remove :meth:`tempfile.SpooledTemporaryFile.softspace` as files no longer have the ``softspace`` attribute in Python 3. Patch by Shantanu.
```

File: Misc/NEWS.d/next/Library/2020-02-21-22-35-55.bpo-39719.2jLy1C.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Remove :meth:`tempfile.SpooledTemporaryFile.softspace` as files no longer have the ``softspace`` attribute Python 3.
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Library/2020-02-21-22-35-55.bpo-39719.2jLy1C.rst]: Please remove the duplicate.

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]
LGTM. Just remove the duplicate of the NEWS entry.

**Comment by hauntsaninja** [comment]
I have made the requested changes; please review again

**Review by serhiy-storchaka — APPROVED** [review]
:+1:

**Pull request merged**